### DISCUSSION TITLE
How do I auth as admin with password?

### DISCUSSION THREAD

**Discussion by DzigiBauk** [pocketbase/pocketbase#1184] [Q&A] [answered]
I've been following [this](https://pocketbase.io/docs/authentication) part of the documentation, but I simply can't get it to work. Pretty much copy-pasted it, and it still comes back with a 404.

The exact error:
![image](https://user-images.githubusercontent.com/44768614/205714969-4173512a-bc5f-4d53-93f8-1865ec16b0ae.png)

I couldn't find anyone with a similar issue, nor could I find a discussion that would elaborate a bit on the matter. I'm probably doing something wrong, so could somebody point me in the right direction?

**Comment by ganigeorgiev** [comment]
Are you sure that you are using the correct PocketBase base url?

What versions of the PocketBase backend and JS SDK are you using?

Could you provide a code sample how do you initialize the JS SDK?

**Comment by DzigiBauk** [comment, marked as answer]
What I linked above (the documentation), is the exact code. I'm using version 0.7.10 (the version that was current at the time when I installed). Currently using NodeJS 18.10.0.

**Reply by ganigeorgiev** [reply]
PocketBase JS SDK v0.8 is compatible only with PocketBase v0.8+. This explains the error, because in v0.8 the api url is different.

You can check the v0.7.x migration guide in https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md#migrate-from-v07x.

**Reply by DzigiBauk** [reply]
Updating fixed it. Thanks.
